scripts/country_removal.py

```diff
@@ -0,0 +1,64 @@
+from invenio_search import current_search_client as es
+from invenio_pidstore.models import PersistentIdentifier
+from invenio_records_files.api import Record
+from invenio_db import db
+import re
+
+
+def get_cern_cooperation_agreement():
+    records = {}
+    query = {
+        "_source": ["_files", "control_number", "dois"],
+        "query": {
+            "regexp": {
+                "authors.affiliations.value": {
+                    "value": ".*cooperation.*",
+                    "flags": "ALL",
+                    "case_insensitive": "true",
+                }
+                }
+            }
+        }
+
+    search_result = es.search(index="scoap3-records-record", scroll='1m', body=query)
+    sid = search_result['_scroll_id']
+    scroll_size = len(search_result['hits']['hits'])
+    while (scroll_size > 0):
+        for record_index in range(scroll_size):
+            recid = search_result["hits"]["hits"][record_index]["_source"]["control_number"]
+            doi = search_result["hits"]["hits"][record_index]["_source"]["dois"][0]["value"]
+            records[recid] = doi
+        search_result = es.scroll(scroll_id=sid, scroll='2m')
+        sid = search_result['_scroll_id']
+        scroll_size = len(search_result['hits']['hits'])
+    return records
+
+def remove_country(record):
+    for author in record["authors"]:
+        try:
+            for affiliation in author["affiliations"]:
+                pattern_for_cern_cooperation_agreement = re.compile(r'cooperation agreement with cern', re.IGNORECASE)
+                match_pattern = pattern_for_cern_cooperation_agreement.search(affiliation["value"])
+                if match_pattern is not None:
+                    affiliation.pop("country")
```

Comment:
```suggestion
                if match_pattern is not None:
                    affiliation.pop("country")
```

I would suggest adding the try/except here so it doesn't fail for the rest of affiliations. Alternatively we could just do:

```python
affiliation.pop('country', None)
```

which doesn't need the `try/catch`